SEED-platform/seed
# Default to excluding Address Line 1 as a matching criteria

#### What's this PR do?
Removes `Address Line 1` as a default column to match on, since it is more error-prone than useful

#### How should this be manually tested?
1. Create a new organization
2. Check the Column Settings page for matching columns, ensure that `Address Line 1` is unchecked
3. Upload the following csv, ensure that the two records get merged automatically

| Address Line 1 | Custom ID 1 |
|---|---|
| 123 Main St | 1 |
| 1428 Elm St | 1 |

#### What are the relevant tickets?
#4255

#### Screenshots (if appropriate)
![image](https://github.com/SEED-platform/seed/assets/411466/989fd02a-b031-4d18-a2e5-cf5ec7996bd5)
